region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -242,7 +255,35 @@
     },
     {
       "link": [2, 1],
-      "name": "Boulder Room Doorsill Jump",
+      "name": "Jump Through the Door Air Ball",
+      "entranceCondition": {
+        "comeInJumping": {
+          "speedBooster": "any",
+          "minTiles": 1
```

Comment:
At 1 tile this is a last frame jump and that is a tricky jump.

Reply:
Good catch. Split it now into two strats, a 2-tile runway version with canCarefulJump and a 1-tile version with canTrickyJump.